src/main/java/fi/aalto/cs/apluscourses/presentation/ModuleSelectionViewModel.java

```diff
@@ -3,6 +3,9 @@
 import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getText;
 
 import com.intellij.openapi.module.Module;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.project.ProjectUtil;
+import com.intellij.openapi.vfs.VirtualFile;
```

Comment:
Ideally, we would like to keep the view model layer free of IntelliJ dependent stuff (but as it can be seen here, it wasn't that to begin with)